docs/source/development/commands_reference.md

```diff
@@ -310,21 +320,23 @@ kedro run
+| `kedro run --to-nodes <node_name1>,<node_name1>`                     | A list of node names which should be used as an end point                                                                                                                                                                                               | No                          |
+| `kedro run --node <node_name1>,<node_name2>`                         | Run only nodes with specified names                                                                                                                                                                                                                     | Yes                         |
+| `kedro run --runner <runner_name>`                                   | Run the pipeline with a specific runner                                                                                                                                                                                                                 | No                          |
+| `kedro run --env <env_name>`                                         | Run the pipeline in the env_name environment. Defaults to local if not provided                                                                                                                                                                         | No                          |
+| `kedro run --tag <tag_name1>,<tag_name2>`                            | Run only nodes which have any of these tags attached                                                                                                                                                                                                    | Yes                         |
+| `kedro run --load-version="<dataset_name>:YYYY-MM-DDThh.mm.ss.sssZ"` | Specify a particular dataset version (timestamp) for loading                                                                                                                                                                                            | Yes                         |
+| `kedro run --pipeline <pipeline_name>`                               | Run the whole pipeline by its name                                                                                                                                                                                                                      | No                          |
+| `kedro run --config <config_file_name>.yml`                          | Specify all command line options in a configuration file called config.yml                                                                                                                                                                              | No                          |
```

Comment:
```suggestion
| `kedro run --config <config_file_name>.yml`                          | Specify all command line options in a named YAML configuration file                                                                                                                                                                              | No                          |
```

Reply:
One minor change since you've changed the name of the config file. I'd either modify the text from `config.yml` -> `config_file_name.yml` or just avoid mentioning the name as in my suggestion, since the user can choose what they want.